### ISSUE TITLE
Question: How do I avoid creating a TODO entry for future data

### ISSUE THREAD

**Issue by yosevu** [bastibe/org-journal#356]
> org-journal-new-scheduled-entry - prompts for a date, and creates a new journal entry on that date with its timestamp set to the date. By default, this is a TODO entry. Set the prefix to avoid the TODO.

Hello, I see the above note in the README, but I have been haven't been able to get it to work yet. How do I prevent the following from being added?

```
TODO 
<2021-06-27 Sun 00:00>
```

Config:

```elisp
(org-journal-file-header "#+title: %Y Journal\n#+created: %Y-%m-%d\n#+filetags: \"journal\"\n\n")
  (org-journal-date-format "[%Y-%m-%d %a %R] - Week %V")
  (org-journal-date-prefix "* ")
  (org-journal-time-format "")
  (org-journal-time-prefix ""))
```

Action: `org-ournal-new-date-entry` with future date.

Result:

<img width="386" alt="Screen Shot 2021-06-26 at 8 46 04 PM" src="https://user-images.githubusercontent.com/16216104/123530408-87c71180-d6bf-11eb-8181-e146a35f261d.png">

**Comment by bastibe** [comment]
If I remember correctly, C-u when creating the entry prevents the TODO and timestamp.

**Comment by yosevu** [comment]
Ah, thanks. I see that in the function doc. It doesn't add the `TODO` anymore. Is there a way to also omit the schedule date?

<img width="294" alt="Screen Shot 2021-07-06 at 11 43 19 PM" src="https://user-images.githubusercontent.com/16216104/124701301-f2dcc900-deb3-11eb-91ac-f8b3eabf8704.png">

**Comment by bastibe** [comment]
A double-prefix (C-u C-u) should prevent the creation of the timestamp as well.

**Comment by yosevu** [comment]
Hmm, I do see the doc in `org-journal-new-date-entry`, but it still seems to only remove the `TODO` after:

1. `C-u C-u`
2. `org-journal-new-date-entry`
3. Select future date
4. Return

<img width="348" alt="Screen Shot 2021-07-18 at 8 58 02 AM" src="https://user-images.githubusercontent.com/16216104/126070050-7f66eb62-d470-4b39-8363-bf64cd108367.png">

What else could I be missing?

**Comment by bastibe** [comment]
Indeed. This seems to be a bug in the documentation. This functionality probably used to work at some time, but got lost during a refactoring. Sorry about that.

I currently don't have the time to look into this in-depth. But if you'd like to take a stab at it, I'll try to help you as best I can.
